src/schema/city/index.ts

```diff
@@ -115,6 +117,25 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           artGuideUrl: {
             type: GraphQLString,
           },
+          shows: {
+            type: showConnection,
+            args: pageable({
+              sort: PartnerShowSorts,
+              status: EventStatus,
+            }),
+            resolve: async (
+              citySponsoredContent,
+              args,
+              { showsWithHeadersLoader }
+            ) => {
+              return loadData(args, showsWithHeadersLoader, {
+                id: citySponsoredContent.showIds,
```

Comment:
nice! so this is why we don't need fetch by `partner_ids` anymore right?

Reply:
Exactly 🎯